styleguide/components/31-molecules/teasers/teaser/teaser.twig

```diff
@@ -0,0 +1,110 @@
+<li class="teaser {{ teaser_type }} {{ modifier }}">
+  <article class="teaser-content">
+    {# title must be the first element in markup #}
+    <div class="content__bottom">
+      <{{ header_level|default('h3') }} id="{{ teaser_uid }}">{{ teaser_title }}</{{ header_level|default('h3') }}>
```

Comment:
There's a possibility to pass in a custom header_level in the template, but only the `h3` tag is styled in the sass file

Reply:
Okay but this is just used in styleguide ... users are not supposed to use h2 or any other heading type here.

Reply:
if no-one can use it, then why keep it?

Reply:
It's legacy. Went like this: 

1. teaser-contact: `<h4><a href="#">Title text</a></h4>`
2. @delrueba said: "No links in titles 🙅‍♂️"
3. I removed the header_level in the parent template, but forgot to remove it in the teaser template 👼

Reply:
Fixed.